libevmcore/Instruction.cpp

```diff
@@ -165,6 +165,7 @@ const std::map<std::string,  Instruction> dev::eth::c_instructions =
 	{ "LOG3", Instruction::LOG3 },
 	{ "LOG4", Instruction::LOG4 },
 	{ "CREATE", Instruction::CREATE },
+	{ "CREATE_P2SH", Instruction::CREATE_P2SH },
```

Comment:
It's renamed now to `CREATE2`, could you rename it everywhere in code? (including `Executive::createP2shOpcode` method)